Source/SwiftLintBuiltInRules/Rules/Style/StatementPositionRuleExamples.swift

```diff
@@ -0,0 +1,322 @@
+struct StatementPositionRuleExamples {
+    static let nonTriggeringExamples = [
+        // Single line examples
+        Example("if true { } else if false { }"),
+        Example("if true { } else { }"),
+        Example("do { } catch { }"),
+        Example("do { let a = 1 } catch let error { }"),
+        Example("\"}else{\""),
+        Example("""
+        struct A { let catchphrase: Int }
+        let a = A(
+         catchphrase: 0
+        )
+        """),
+        Example("""
+        struct A { let `catch`: Int }
+        let a = A(
+         `catch`: 0
+        )
+        """),
+        // Multi-line examples
+        Example("""
+        if true {
+            foo()
+        } else {
+            bar()
+        }
+        """),
+        Example("""
+        if true {
+            foo
+        } else if true {
+            bar()
+        } else {
+            return
+        }
+        """),
+        Example("""
+        do {
+            foo()
+        } catch {
+            bar()
+        }
+        """),
+        Example("""
+        do {
+            foo()
+        } catch let error {
+            bar()
+        } catch {
+            return
+        }
+        """),
+        Example("""
+        struct A { let catchphrase: Int }
+        let a = A(
+            catchphrase: 0
+        )
+        """),
+        Example("""
+        struct A { let `catch`: Int }
+        let a = A(
+            `catch`: 0
+        )
+        """),
+    ]
+
+    static let triggeringExamples = [
+        // Single line examples
+        Example("if true { ↓}else if false { }"),
+        Example("if true { ↓}  else { }"),
+        Example("""
+        do { ↓}
+        catch { }
+        """),
+        Example("""
+        do {
+            let a = 1
+        ↓}
+        \t  catch { }
+        """),
+        // Multi-line examples
+        Example("""
+        if true {
+            foo()
+        ↓}else {
+            bar()
+        }
+        """),
+        Example("""
+        if true {
+            foo()
+        ↓}    else if true {
+            bar()
+        }
+        """),
+        Example("""
+        if true {
+            foo()
+        ↓}
+        else {
+            bar()
+        }
+        """),
+        Example("""
+        do {
+            foo()
+        ↓}catch {
+            bar()
+        }
+        """),
+        Example("""
+        do {
+            foo()
+        ↓}    catch {
+            bar()
+        }
+        """),
+        Example("""
+        do {
+            foo()
+        ↓}
+        catch {
+            bar()
+        }
+        """),
+    ]
+
+    static let corrections = [
```

Comment:
Misses examples that proof that corrections aren't applied if there are between closing brace and keyword.

Reply:
Our correction infra for generated tests doesn't support corrections that are skipped, but I'll add some triggering examples that wouldn't be corrected.

Reply:
In these cases, I typically add a correction pair where both examples are equal to show and test that nothing changes.